at_commons/lib/src/verb/sync_verb_builder.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:at_commons/at_builders.dart';
+
+class SyncVerbBuilder implements VerbBuilder {
+  int? commitId;
```

Comment:
commitId is non null param for sync. you can mark as late.

Reply:
Marked commitId as late var commitId